src/docs/blueprint_documentation.md

```diff
@@ -113,8 +113,21 @@ In addition, some parts of the installer can be customized.
 #### Application Window Title
 
 The window title is the application’s name if no workspace is opened and `<workspace name> — <application name>` if a workspace is opened.
-The application name can be adapted in `applications/electron/package.json`:
-Open the file and adapt the value of property `theia.frontend.config.applicationName` to the desired name.
+To clarify that the property is nested in `package.json`, you can explicitly mention the hierarchy and emphasize the nesting structure in your documentation. Here's an improved version:
+
+---
+
+The application name can be adapted in `applications/electron/package.json` by navigating through the nested properties.
+
+To change the application name, open the file and locate the following nested structure:
+
+- `theia`
```

Comment:
I wonder if it might look nicer if we nested this in a ```typescript block, like they did under "Customizing the Welcome Page": https://eclipse-theia.github.io/theia-website-previews/pr-previews/pr-613/docs/blueprint_documentation/#application-window-title

Reply:
you mean with ```json?
I do not see the example you mean under "welcome page"